### ISSUE TITLE
A Question About Extracted Data

### ISSUE THREAD

**Issue by 2321281653** [lcd-dal/feature-extraction-for-CERT-insider-threat-test-dataset#11]
Hello, I am a researcher from China. I've run the feature_extraction.py and got the file dayr4.2.csv, but the insider columns in it all have values of 0. Is this the correct result? Is there any missing anomalous data with an insider column value of 1?

**Comment by lcd-dal** [comment]
there should be other values indicating the insider threat scenario.
Did you extract the whole data? I think there was some days that contain no insiders

**Comment by 2321281653** [comment]
> there should be other values indicating the insider threat scenario. Did you extract the whole data? I think there was some days that contain no insiders

Yes, I have extracted all the data. The results of multiple runs are consistent. Perhaps I should try switching to the r5.2 dataset instead?

**Comment by lcd-dal** [comment]
Hard to tell what went wrong in your case. I could provide extracted r4.2 data if you want
For r5.2, you can download pre-extracted data here: https://web.cs.dal.ca/~lcd/data/CERTr5.2/
If you'd like to run the extraction yourself, it should output the same data.

**Comment by 2321281653** [comment]
> extracted r4.2 data

Yes, I'm in urgent need of the extracted data of dayr4.2 and weekr4.2. I'm truly grateful that you're willing to provide them to me. My email is 2321281653@qq.com.

**Comment by lcd-dal** [comment]
here are the links:
CERT r5.2: https://web.cs.dal.ca/~lcd/data/CERTr5.2/
r4.2: https://web.cs.dal.ca/~lcd/data/CERTr4.2/dayr4.2.csv.zip, https://web.cs.dal.ca/~lcd/data/CERTr4.2/weekr4.2.csv.zip

**Comment by 2321281653** [comment]
> here are the links: CERT r5.2: https://web.cs.dal.ca/~lcd/data/CERTr5.2/ r4.2: https://web.cs.dal.ca/~lcd/data/CERTr4.2/dayr4.2.csv.zip, https://web.cs.dal.ca/~lcd/data/CERTr4.2/weekr4.2.csv.zip

I'm deeply indebted to you for your help!
